HISTORY.md

```diff
@@ -1,6 +1,6 @@
 # History
 
-# 1.3.1 (2022-12-20)
+## 1.3.1 (2022-12-20)
 
```

Comment:
Should probably be a separate PR for clean up as unrelated to 3.7

Reply:
I will update dependencies in a separate PR but keeping this change in this PR as this is one line only.